.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - restore_cache:
           name: Restore Homebrew + Ruby Dependencies
           keys:
-            - &cache_key brew-dependencies-{{ checksum ".circleci/.brewfile" }}-{{ checksum "Gemfile.lock" }}
+            - &cache_key brew-dependencies-{{ checksum ".circleci/cache-version" }}-{{ checksum ".circleci/.brewfile" }}-{{ checksum "Gemfile.lock" }}
```

Comment:
It looks like `imagemagick@6` formulae got updated very recently and it broke the installation on CI because the previous version was in the cache and `homebrew` didn't know which one to link. Interestingly, `homebrew` defaults to returning error code 1 in this case. 

I'm pretty sure there are smarter things we can do with how we manage the CircleCI cache, but, since we're planning to switch away from CircleCI and, so, spending more time on it may not be worth it, I opted for implementing the solution we've used on WPiOS for a few years now.